# Header
std_msgs/Header header

#Data
landmark[] local_landmarks
landmark[] world_landmarks